resolwe_bio/tools/parse_diffexp.py

```diff
@@ -34,8 +35,12 @@ def main():
     col_order = []
 
     if args.gene_id:
-        columns['gene_id'] = list(de_data[args.gene_id])
-        col_order.append('gene_id')
+        if args.gene_id == 'index':
```

Comment:
This is to handle the way how GeneIDs get parsed from the DE results file. Some DE tools return tab-delimited files where column containing geneIDs is not labeled with the column name. When parsing such files, in some versions of the DE ouput files such column is returned as "Unnamed column 0" and sometimes as index column.